employee-rostering-frontend/package-lock.json

```diff
@@ -6724,9 +6724,9 @@
       }
     },
     "eslint-utils": {
-      "version": "1.4.0",
-      "resolved": "https://registry.npmjs.org/eslint-utils/-/eslint-utils-1.4.0.tgz",
-      "integrity": "sha512-7ehnzPaP5IIEh1r1tkjuIrxqhNkzUJa9z3R92tLJdZIVdWaczEhr3EbhGtsMrVxi1KeR8qA7Off6SWc5WNQqyQ==",
+      "version": "1.4.2",
+      "resolved": "https://repository.engineering.redhat.com/nexus/repository/registry.npmjs.org/eslint-utils/-/eslint-utils-1.4.2.tgz",
```

Comment:
No references to Red Hat internal registry, please.

Reply:
Done, although it was VERY annoying to get it to use the public registry. I made the following bash script to do it without modifying my `.npmrc`:

```
#!/bin/sh
npm_config_cache=$(mktemp -d)
rm package-lock.json
rm -rf node_modules
npm install --cache=$npm_config_cache --registry=https://registry.npmjs.org
rm -rf $npm_config_cache
```

Reply:
Yes. When I realized this I reverted my `.npmrc` to not use redhat registry. I think it was bad idea to use it on a dev machine. And yes, I also needed to remove node_modules to get rid of it.